src/app/admin/clients/page.tsx

```
@@ -363,69 +378,31 @@ export default function AdminClientsPage() {
-                    ].join('\n')
-
-                    // Create and trigger download
-                    const blob = new Blob([csvContent], { type: 'text/csv;charset=utf-8;' })
-                    const link = document.createElement('a')
-                    link.href = URL.createObjectURL(blob)
-                    link.download = `clients_export_${new Date().toISOString().split('T')[0]}.csv`
-                    link.click()
-                    URL.revokeObjectURL(link.href)
-
+                // Generate CSV content for selected clients
+                case 'export': {
```

Biome (1.9.4)

[error] Duplicate case label.

The first similar label is here:


(lint/suspicious/noDuplicateCase)